nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -236,6 +236,16 @@
      release announcement</link> for more information.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     Restya-Board has been upgraded from 6.0 to 6.7, improving security and
+     adding features but with potential incompatibilities.
+    </para>
+    <para>
+     Please note: When upgrading this software, you have to run the database
+     migration scripts manually. You can find them in the source repository
+     under /sql.
+   </listitem>
```

Comment:
```suggestion
    </para>
   </listitem>
```

Reply:
ok. sry. I added the misssing closing tag.